var/spack/repos/builtin/packages/r-adabag/package.py

```diff
@@ -9,12 +9,15 @@
 class RAdabag(RPackage):
     """Applies Multiclass AdaBoost.M1, SAMME and Bagging."""
 
-    homepage = "https://cran.r-project.org/package=adabag"
-    url      = "https://cran.r-project.org/src/contrib/adabag_4.1.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/adabag"
+    homepage = "https://cloud.r-project.org/package=adabag"
+    url      = "https://cloud.r-project.org/src/contrib/adabag_4.1.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/adabag"
 
+    version('4.2', sha256='47019eb8cefc8372996fbb2642f64d4a91d7cedc192690a8d8be6e7e03cd3c81')
     version('4.1', '2e019f053d49f62ebb3b1697bbb50afa')
 
     depends_on('r-rpart', type=('build', 'run'))
     depends_on('r-mlbench', type=('build', 'run'))
```

Comment:
This dependency is suggested, not required. I would remove this.

Reply:
I too noticed that some packages had included "suggested" packages. My first instinct was to remove them but then I was not sure if the package author had simply overdid the dependency specs or if there was a good reason to include those. I left them in to avoid potentially breaking something. My feeling is that the dependencies were simply over stated so the risk to breakage is probably small.

Reply:
This actually depends on mlbench up to v4.1. I Made that change.